### PR TITLE
Update the README.md

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
As the title says.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [X] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by theomoura** [comment]
Add Changelog

File: README.md [hunk 1]
```diff
@@ -9,30 +12,204 @@ Search Result is a canonical component that any VTEX store can install.
...
+
+Also, you can configure the product summary that is defined on minicart. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API. 
+
+### Styles API
+This app provides some CSS classes as an API for style customization.
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | [SearchResult](https://github.com/vtex-apps/search-result/blob/master/react/components/SearchResult.js) | The main container of search-result |
+| `buttonShowMore`        | [ShowMoreLoaderResult](https://github.com/vtex-apps/search-result/blob/master/react/components/loaders/ShowMoreLoaderResult.js) | Show the see more button |
```
Comment (theomoura) [inline README.md]: The description, in this case, can't be an action 

```suggestion
| `buttonShowMore`        | [ShowMoreLoaderResult](https://github.com/vtex-apps/search-result/blob/master/react/components/loaders/ShowMoreLoaderResult.js) | The "Show more" button in ... |
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-# Search Result
+# VTEX Search Result
 
-Search Result is a canonical component that any VTEX store can install.
+## Description
+The VTEX Search Result app handles the result of a search and this app is used by the Dreamstore product.
```
Comment (brunojdo) [inline README.md]: The VTEX Search Result app is a store component that handles ...

Comment (brunojdo) [reply README.md]: ... by store theme.


Please check the last doc of the Minicart

File: README.md [hunk 3]
```diff
@@ -9,30 +12,229 @@ Search Result is a canonical component that any VTEX store can install.
...
+}
+```
+
+#### CSS namespaces
+
+Below, we describe the namespaces that are defined in the minicart.
+
+| Token name         | Description                   | Component Source |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`         | The main container of search-result | [SearchResult](https://github.com/vtex-apps/search-result/blob/master/react/components/SearchResult.js) |
```
Comment (theomoura) [inline README.md]: You can use only "/react/componente/SearchResult.js" on Thais links

File: README.md [hunk 4]
```diff
@@ -1,38 +1,239 @@
-# Search Result
+# VTEX Search Result
 
-Search Result is a canonical component that any VTEX store can install.
+## Description
+The VTEX Search Result app is a store component that handles the result of a search, and this app is used by store theme.
```
Comment (brunojdo) [inline README.md]:
```suggestion
The VTEX Search Result app is a store component that handles with the result of our [_Search API_](https://documenter.getpostman.com/view/845/vtex-search-api/Hs43#8b71745e-00f9-6c98-b776-f4468ecb7a5e), and this app is used by store theme.
```

File: README.md [hunk 5]
```diff
@@ -1,38 +1,239 @@
...
+      "order-by"
+    ],
+    "required": [
+      "gallery"
+    ],
+    "component": "index"
+  },
+```
+
+The search-result has as a required block the `gallery`. So, any search-result block implementation created must add a gallery as a block that is inside of search-result. Similarly, `gallery` has its own inner block structure that can be configured that you can see below.
```
Comment (theomoura) [inline README.md]:
```suggestion
The search-result has as a required block the `gallery`. So, any search-result block implementation created must add a gallery as a block that is inside of search-result. Similarly, `gallery` has its own inner block structure that can be configured like you can see below.
```

File: README.md [hunk 6]
```diff
@@ -1,38 +1,239 @@
...
+| `queryField`          | `String`  | Query field                             | N/A
+| `mapField`              | `String`  | Map field                                                  | N/A |
+| `restField`                   | `String`  | Other Query Strings       | N/A |
+| `orderByField`                   | `String`  | Order by field (values: 'OrderByTopSaleDESC', 'OrderByReleaseDateDESC', 'OrderByBestDiscountDESC', 'OrderByPriceDESC', 'OrderByPriceASC', 'OrderByNameASC' or 'OrderByNameDESC')       | `OrderByReleaseDateDESC` |
+ 
+HiddenFacets
+
+| Prop name          | Type       | Description   | Default value |
+| ------------------ | ---------- | --------------------------------------------------------------------------- | --- |
+| `layoutMode1`            | `String`  | Layout mode of the switcher (values: 'normal', 'small' or 'inline')    | `normal` |
```
Comment (theomoura) [inline README.md]: Isn't this a `Enum` type?

File: README.md [hunk 7]
```diff
@@ -1,38 +1,239 @@
...
+        "maxItemsPerPage": 2,
+        "orderByField": "OrderByReleaseDateDESC"
+      },
+      "hiddenFacets": {
+        "layoutMode1": "normal",
+        "layoutMode2": "small",
+        "specificationFilters": {
+          "hiddenFilters": [
+            {
+              "__editorItemTitle": "editor.search-result.hiddenFacets.specificationFilters.hiddenFilter"
```
Comment (brunojdo) [inline README.md]: remove this. It is just used by Storefront. If you don't add an example of `hiddenFilters`, remove it

File: README.md [hunk 8]
```diff
@@ -1,38 +1,239 @@
...
+            }
+          ]
+        }
+      },
+      "summary": {
```
Comment (brunojdo) [inline README.md]: remove this too.

File: README.md [hunk 9]
```diff
@@ -1,38 +1,239 @@
...
+
+```css
+.container {
+  margin-top: 10px;
+}
+```
+
+#### CSS namespaces
+
+Below, we describe the namespaces that are defined in the minicart.
```
Comment (brunojdo) [inline README.md]: search-result

File: README.md (hunk 6 above)
Comment (vitoriaHeliane) [reply README.md]: No, it is a String. :smile:

Comment (theomoura) [reply README.md]: Isn't is a `Enum` of type String?

In this case 
```suggestion
| `layoutMode1`            | `Enum`  | Layout mode of the switcher (Possible values: 'normal', 'small' or 'inline')    | `normal` |
```

File: README.md [hunk 10]
```diff
@@ -1,38 +1,219 @@
...
 
-```sh
-$ npm t     # this
-$ npm test  # or
+QuerySchema
+
+| Prop name          | Type       | Description  | Default value |
+| ------------------ | ---------- | --------------------------------------------------------------------------- | --- |
+| `maxItemsPerPage`   | `Number`   | Maximum number of items per search page                     | 10 |
+| `queryField`          | `String`  | Query field                             | N/A
```
Comment (theomoura) [inline README.md]: For conciseness use:

```suggestion
| `queryField`          | `String`  | Query field                             | -
```

**Pull request merged**